include/cloysterhpc/services/IService.h

```diff
@@ -8,12 +8,14 @@
 
 class IService {
 protected:
+    [[maybe_unused]] virtual void configure() = 0;
+
+public:
+    [[maybe_unused]] virtual void install() = 0;
```

Comment:
If it's a virtual void function does it make sense to be ``[[maybe_unused]]``? I don't get it.

Reply:
As an example, the last PR approved that added ITool used install. This one doesn't..